store/tikv/delete_range.go

```diff
@@ -77,13 +77,13 @@ func (t *DeleteRangeTask) Execute(ctx context.Context) error {
 }
 
 // Execute performs the delete range operation.
-func (t *DeleteRangeTask) sendReqOnRange(ctx context.Context, r kv.KeyRange) (int, error) {
+func (t *DeleteRangeTask) sendReqOnRange(ctx context.Context, r kv.KeyRange) (RangeTaskStat, error) {
 	startKey, rangeEndKey := r.StartKey, r.EndKey
-	completedRegions := 0
+	var stat RangeTaskStat
```

Comment:
I do not think to define a variable without init it is a good idea, is it a good habit in golang?

Reply:
It's field should be initialized as 0 by default here.

Reply:
Totally good habit in Go.